legacy/gen/fontfixed.inc

```diff
@@ -1,19 +1,51 @@
+	/* 0x00 _ */ (uint8_t *)"\x01\x00",
+	/* 0x01 _ */ (uint8_t *)"\x01\x00",
+	/* 0x02 _ */ (uint8_t *)"\x01\x00",
+	/* 0x03 _ */ (uint8_t *)"\x01\xff",
+	/* 0x04 _ */ (uint8_t *)"\x01\x00",
+	/* 0x05 _ */ (uint8_t *)"\x01\x00",
+	/* 0x06 _ */ (uint8_t *)"\x07\x18\x1c\x0e\x18\x30\x40\x80",
+	/* 0x07 _ */ (uint8_t *)"\x01\x00",
+	/* 0x08 _ */ (uint8_t *)"\x09\x10\x28\x44\x82\xaa\x92\xaa\x82\xfe",
+	/* 0x09 _ */ (uint8_t *)"\x05\x0e\x02\x02\x02\x0e",
+	/* 0x0a _ */ (uint8_t *)"\x01\x00",
+	/* 0x0b _ */ (uint8_t *)"\x01\x00",
+	/* 0x0c _ */ (uint8_t *)"\x01\x00",
+	/* 0x0d _ */ (uint8_t *)"\x01\x00",
+	/* 0x0e _ */ (uint8_t *)"\x01\x00",
+	/* 0x0f _ */ (uint8_t *)"\x01\x00",
+	/* 0x10 _ */ (uint8_t *)"\x01\x00",
+	/* 0x11 _ */ (uint8_t *)"\x01\x00",
+	/* 0x12 _ */ (uint8_t *)"\x01\x00",
+	/* 0x13 _ */ (uint8_t *)"\x01\x00",
+	/* 0x14 _ */ (uint8_t *)"\x01\x00",
+	/* 0x15 _ */ (uint8_t *)"\x07\x44\xee\x7c\x38\x7c\xee\x44",
+	/* 0x16 _ */ (uint8_t *)"\x01\x00",
+	/* 0x17 _ */ (uint8_t *)"\x01\x00",
+	/* 0x18 _ */ (uint8_t *)"\x01\x00",
+	/* 0x19 _ */ (uint8_t *)"\x01\x00",
+	/* 0x1a _ */ (uint8_t *)"\x01\x00",
+	/* 0x1b _ */ (uint8_t *)"\x01\x00",
+	/* 0x1c _ */ (uint8_t *)"\x01\x00",
+	/* 0x1d _ */ (uint8_t *)"\x01\x00",
+	/* 0x1e _ */ (uint8_t *)"\x01\x00",
+	/* 0x1f _ */ (uint8_t *)"\x01\x00",
 	/* 0x20   */ (uint8_t *)"\x01\x00",
-	/* 0x21 ! */ (uint8_t *)"\x03\x60\xfa\x60",
+	/* 0x21 ! */ (uint8_t *)"\x05\x00\x60\xfa\x60\x00",
```

Comment:
Can you please edit `fontfixed.png` accordingly? This file is generated from this image, so the png file should match.